gmso/utils/units.py

```diff
@@ -79,12 +76,426 @@ def default_reg():
             A unyt registry with commonly used conversions defined.
         """
         reg = u.UnitRegistry()
-        conversion = (
-            1 * getattr(u.physical_constants, "elementary_charge").value
-        )
-        dimensionsList = [u.dimensions.current_mks, u.dimensions.time]
-        dim = np.prod(dimensionsList)
-        name = "elementary_charge"
-        symbol = r"\rm{e}"
-        reg.add(name, conversion, dim, symbol)
+        register_general_units(reg)
         return reg
+
+
+@staticmethod
```

Comment:
```suggestion
```

Reply:
Fixed in 84dc45819b2d4e50a17f07eda7ba7e0f80d0d33a